.mk/k8s.mk

```diff
@@ -37,7 +37,7 @@ include .mk/null.mk
 
 include .mk/gke.mk
 
-SPIRE_ENABLED?=true
+SPIRE_ENABLED?=false
```

Comment:
Curious why we are defaulting to SPIRE_ENABLED=false ?

Reply:
I didn't know that a `spire-install` make action exists so I think it's better to change the documentation mentioning this action.

Reply:
I'm starting to feel like perhaps our issues are in some gaps between our Makefiles and the helm charts.  @denis-tingajkin  @haiodo  you've looked at this more recently than I have, thoughts?

Reply:
spire is enabled by default, I think we should keep this behaviour